config/client.json

```diff
@@ -26,5 +26,7 @@
   "siftscience_key",
   "facebook_api_key",
   "discover_blog_id",
-  "support-user"
+  "support-user",
+  "sync-handler",
```

Comment:
It's unnecessary to add `sync-handler` here, as it's just a subset of `features` which is already shared with the client.